app/utilities/githubApi/index.js

```diff
@@ -8,10 +8,28 @@ import Notifier from '../notifier'
 import { remote } from 'electron'
 const logger = remote.getGlobal('logger')
 
+// TODO: move to global configuration
```

Comment:
This can be done after resolve #86 (setting panel & global configuration mechanism).